components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/handler/UserOperationEventHookHandler.java

```diff
@@ -133,7 +135,28 @@ public void handleEvent(Event event) throws IdentityEventException {
                 SecurityEventTokenPayload securityEventTokenPayload = EventHookHandlerUtils
                         .buildSecurityEventToken(eventPayload, eventUri);
                 EventHookHandlerUtils.publishEventPayload(securityEventTokenPayload, tenantDomain, eventUri);
-            }
+            } else if (IdentityEventConstants.Event.POST_ADD_NEW_PASSWORD.equals(event.getEventName()) &&
+                    userOperationEventPublisherConfig.isPublishEnabled()) {
```

Comment:
In a password update is it the same event gets triggered

Reply:
This config is baed on below, Where the same event handler key is shared for all events among the event category. Event is uniquely identified by its name as POST_ADD_NEW_PASSWORD

```
   if ((Constants.EventHandlerKey.WSO2.POST_UPDATE_USER_CREDENTIAL.equals(attribute.getKey()) &&
                eventName.equals(IdentityEventConstants.Event.POST_ADD_NEW_PASSWORD))) {
            return true;
        }
```